codeSnippets/snippets/static-resources/src/main/kotlin/com/example/Application.kt

```diff
@@ -3,14 +3,18 @@ package com.example
 import io.ktor.server.application.*
 import io.ktor.server.http.content.*
 import io.ktor.server.routing.*
-import java.io.File
-
 
 fun Application.module() {
     routing {
-        staticResources("/", "static") {
-            default("index.html")
+        /*
+        staticResources("/", "static")
+        staticResources("/", "static", index = "custom_index.html")
+         */
+        staticResources("/", "static"){
+            extensions("html", "htm")
+            enableAutoHeadResponse()
             preCompressed(CompressedFileType.GZIP)
```

Comment:
there are no pre compressed files in the resources folder

Reply:
isn't `js/script.js.gz` ?

Reply:
sorry, missed it